src/main.py

```
@@ -2,47 +2,51 @@
-    parser.add_argument(
-        '--output',
-        default=os.path.join("data", "output", "croco.jpg"),
-        help='Path to output image'
-    )
+    parser = argparse.ArgumentParser(description='Process images with filters and math utils.')
+    parser.add_argument('--input', default=os.path.join("data", "input", "croco.jpeg"), help='Path to input image')
+    parser.add_argument('--output', default=os.path.join("data", "output", "croco.jpg"), help='Path to output image')
     args = parser.parse_args()
 
     input_path = args.input
     output_path = args.output
```

Ruff (0.8.2)

Local variable `output_path` is assigned to but never used

Remove assignment to unused variable `output_path`

(F841)